source/components/molecules/GroupedList/GroupedList.tsx

```diff
@@ -23,14 +23,30 @@ const ListBodyFieldLabel = styled(Label)`
   margin-top: 5px;
 `;
 
```

Comment:
I don't mind these changes, but i would rather have them as a fix PR. In general i would say that the smaller a PR is the easier it is to review, and makes it easier to get things from review to done. Trying to improve this myself aswell (y)

A good overview of how to think of PR's [https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests](can be found here)